frontend/src/features/util.js

```diff
@@ -66,3 +66,5 @@ export function obtainRarityEdge(rarity) {
   if (!rarity) return "";
   return `rarity-${rarity.toLowerCase()}-border`;
 }
+
+export const owners = "fi-apprentice";
```

Comment:
This is a placeholder. Users belonging to the group name mentioned here (ideally, `sysadmin-badges`) would have access to the admin page (i.e. the `/governor/` route).